### ISSUE TITLE
vscode not using accentuation anymore

### ISSUE THREAD

**Issue by alansenairj** [microsoft/vscode#126170]
Issue Type: <b>Bug</b>

1. open vscode using pt-br system base localization and keyboard layout too
2. any dead keys are working. type (^'`)  in terminal and text editor and this dead keys cannot work as expected . 
![image](https://user-images.githubusercontent.com/20565821/121810124-f2933a00-cc35-11eb-866c-a56cb53d13e5.png)

VS Code version: Code 1.57.0 (b4c1bd0a9b03c749ea011b06c6d2676c8091a70c, 2021-06-09T17:18:42.895Z)
OS version: Linux x64 5.11.0-18-generic snap
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i9-9900 CPU @ 3.10GHz (16 x 3100)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>rasterization: disabled_software<br>skia_renderer: enabled_on<br>video_decode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|0, 0, 0|
|Memory (System)|31.28GB (12.67GB free)|
|Process Argv|--force-user-env --unity-launch --crash-reporter-id 54e6c774-53fb-4c6e-9bc2-8d5a087e9af3|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|ubuntu|
|XDG_CURRENT_DESKTOP|Unity|
|XDG_SESSION_DESKTOP|ubuntu|
|XDG_SESSION_TYPE|x11|
</details><details><summary>Extensions (10)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-django|bat|1.6.0
python-extension-pack|don|1.6.0
MagicPython|mag|1.1.0
python|ms-|2021.5.926500501
vscode-pylance|ms-|2021.6.1
jupyter|ms-|2021.6.999230701
vscode-yaml|red|0.19.2
vscodeintellicode|Vis|1.2.14
jinja|who|0.0.8
vscode-ansible|zbr|0.4.5

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
pythonvspyt602:30300191
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyt639:30300192
pythontb:30283811
pythonvspyt551cf:30311713
vspre833:30267464
pythonptprofiler:30281270
vshan820:30294714
pythondataviewer:30285071
vscus158cf:30286554
pythonvsuse255cf:30319632
vscgsv2:30307504
vscorehovct:30309550
vscod805cf:30301675
binariesv517cf:30312826

```

</details>

**Comment by alansenairj** [comment]
I don't know if it is some bug from Linux GTK. I'm using Ubuntu 21.04. My keyboard layout is UTF-8 PTBR.

**Comment by alexdima** [comment]
@alansenairj Could you please try a native `<input>` like the one of the find widget. Can you type accents as expected there?

<img width="562" alt="Screen Shot 2021-06-14 at 17 47 31" src="https://user-images.githubusercontent.com/5047891/121920760-a250d080-cd38-11eb-9a91-f80ab622d56e.png">

**Comment by alansenairj** [comment]
> @alansenairj Could you please try a native `<input>` like the one of the find widget. Can you type accents as expected there?
> 
> <img alt="Screen Shot 2021-06-14 at 17 47 31" width="562" src="https://user-images.githubusercontent.com/5047891/121920760-a250d080-cd38-11eb-9a91-f80ab622d56e.png">

it is not working too

**Comment by MateusMarcuzzo** [comment]
I am having this issue just on the terminal.

**Comment by alansenairj** [comment]
Hi guys, I solved my problem here on Ubuntu 21.04. It was a problem about input method. I just choose IBus here and now everything is working fine. 

![image](https://user-images.githubusercontent.com/20565821/132868224-393ea1a1-c126-4beb-821e-9d2ba258a35a.png)

I hope you guys can solve it too.

**Comment by MateusMarcuzzo** [comment]
> Hi guys, I solved my problem here on Ubuntu 21.04. It was a problem about input method. I just choose IBus here and now everything is working fine.
> 
> ![image](https://user-images.githubusercontent.com/20565821/132868224-393ea1a1-c126-4beb-821e-9d2ba258a35a.png)
> 
> I hope you guys can solve it too.

Nice!

How do you go to this screen configuration? I'm using vscode with english language.

I tried using ctrl+T and navegating through the options, but couldn't find it.
Also the settings window.

Thanks
